src/mmw/js/src/modeling/models.js

```diff
@@ -80,7 +82,127 @@ var ResultModel = Backbone.Model.extend({
-    }
+    },
+
+    toTR55RunoffCSV: function(isCurrentConditions, aoiVolumeModel) {
+        var rows = ['"Runoff Partition","Water Depth (cm)","Water Volume (m3)"'],
+            resultKey = isCurrentConditions ? 'unmodified' : 'modified',
+            result = this.get('result')['runoff'][resultKey],
+            labels = [['runoff', 'Runoff'],
+                      ['et'    , 'Evapotranspiration'],
+                      ['inf'   , 'Infiltration']];
+
+        return rows.concat(labels.map(function(label) {
+            var key = label[0],
+                partition = label[1],
+                depth = result[key],
+                volume = aoiVolumeModel.adjust(depth),
+                row = [
+                        partition,
+                        round(depth, 3),
+                        round(volume, 2)
+                    ].join('","');
+
+            return '"' + row + '"';
+        })).join('\n');
+    },
+
+    toTR55WaterQualityCSV: function(isCurrentConditions, aoiVolumeModel) {
+        var rows = ['"Quality Measure","Load (kg)","Loading Rate (kg/ha)","Average Concentration (mg/L)"'],
+            resultKey = isCurrentConditions ? 'unmodified' : 'modified',
+            results = this.get('result')['quality'][resultKey];
+
+        return rows.concat(results.map(function(result) {
+            var load = result.load,
+                loadingRate = aoiVolumeModel.getLoadingRate(load),
+                adjustedRunoff = aoiVolumeModel.adjust(result.runoff),
+                concentration = adjustedRunoff ? load / adjustedRunoff : 0,
+                row = [
+                        result.measure,
+                        round(load, 3),
+                        round(loadingRate, 3),
+                        round(concentration, 1)
+                    ].join('","');
+
+            return '"' + row + '"';
+        })).join('\n');
+    },
+
+    toMapShedHydrologyCSV: function() {
+        var rows = ['"Month","Precip (cm)","ET (cm)","Surface Runoff (cm)","Subsurface Flow (cm)","Point Src Flow (cm)","Stream Flow (cm)"'],
+            runoffVars = [
+                    'AvPrecipitation',
+                    'AvEvapoTrans',
+                    'AvRunoff',
+                    'AvGroundWater',
+                    'AvPtSrcFlow',
+                    'AvStreamFlow',
+                ],
+            monthNames = [
+                    'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+                    'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+                ],
+            results = this.get('result')['monthly'];
+
+        // Monthly rows
+        rows = rows.concat(results.map(function(result, i) {
+            var cols = [monthNames[i]].concat(runoffVars.map(function(runoffVar) {
+                    return round(result[runoffVar], 2);
+                })).join('","');
+
+            return '"' + cols + '"';
+        }));
+
+        // Total row
+        var totalCols = ['"Total"'].concat(runoffVars.map(function(runoffVar) {
```

Comment:
This is creating an extra quotation mark somewhere:
`""Total"","103.77","50.97","21.27","33.14","81.99","136.39"`
vs 
`"Dec","8.81","1.27","2.61","3.61","6.96","13.19"`